app/assets/stylesheets/common/admin/admin_table.scss

```diff
@@ -1,13 +1,11 @@
 .d-admin-table {
-  box-shadow: none;
```

Comment:
are the three non-breakpoint changes intentionally included? 😃

Reply:
Yes 😄